frontends/pytorch/e2e_testing/torchscript/basic.py

```diff
@@ -0,0 +1,69 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+
+from torch_mlir.torchscript.e2e_test.framework import TestUtils
+from torch_mlir.torchscript.e2e_test.registry import register_test_case
+from torch_mlir.torchscript.annotations import annotate_args, export
+
+# ==============================================================================
+
+class MmModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    @export
+    @annotate_args([
+        None,
```

Comment:
I wonder if it is worth having further sugar for an `@annotate_method` which does not require annotating the `self` arg... fine to let it sit as is and elaborate later.

Reply:
I would like to wait a bit and see what makes the most sense (we might want an all-in-one `export([...shapes...])` or something).